### ISSUE TITLE
[Feature request] Disable Attrition effect via a gamerule

### ISSUE THREAD

**Issue by kristibektashi** [Ladysnake/Requiem#619]

**Comment by PsychoBreadstick** [comment]
I am in support of this request fyi, good idea.

**Comment by James103** [comment]
Workaround: In a repeating command block in spawn chunks run `effect clear @a requiem:attrition`.

**Comment by Eman-3600** [comment]
Probably a good idea, though keep in mind attrition is meant to balance the mod. If you still want to retain some balance, I made an addon mod called [Requiem Lifeline](https://www.curseforge.com/minecraft/mc-mods/requiem-lifeline) that makes attrition a bit more manageable in the early game.

**Comment by Eman-3600** [comment]
Actually, even better idea is a max attrition gamerule. Attrition 4 can be a bit much, so a gamerule to reduce that might make it more manageable.

**Comment by PsychoBreadstick** [comment]
So something like /gamerule requiem:maxAttrition (0-255) with 0 disabling it entirely and each number above zero being the effect’s level?

**Comment by Eman-3600** [comment]
Yeah, though attrition doesn't actually have additional effects beyond level 4 so really just 0-4.

**Comment by PsychoBreadstick** [comment]
Well really, -1 would likely disable it since effect level 0 is basically just effect level 1.

**Comment by PsychoBreadstick** [comment]
Alright.

**Comment by Eman-3600** [comment]
> Well really, -1 would likely disable it since effect level 0 is basically just effect level 1.

Uhhh, this is gonna display in the world creation menu. Let's stick to the displayed level numbers, not the technical ones.

**Comment by PsychoBreadstick** [comment]
I know, I sent that before remembering how attrition works effect-wise and how it would function gamerule-wise.
